src/Graphics/src/Graphics/Text/XmlAttributedTextWriter.cs

```diff
@@ -23,7 +23,7 @@ public string Write(IAttributedText text)
 		{
 			if (attributedText != null && !string.IsNullOrEmpty(attributedText.Text))
 			{
-				bool encode = attributedText.Text.Contains("]]");
+				bool encode = attributedText.Text.IndexOf("]]", StringComparison.Ordinal) != -1;
```

Comment:
Switching to `IndexOf()` for this one felt better than having to do `#if NETSTANDARD2_0`.